src/components/molecules/input/ms-input.stories.ts

```diff
@@ -0,0 +1,36 @@
+import type { Meta, StoryObj } from '@storybook/web-components'
+import { html } from 'lit'
+
+import './ms-input'
+
+export default {
+  title: 'Input',
+  argTypes: {
+    value: { defaultValue: '' },
+    placeholder: { defaultValue: '' },
+    disabled: { options: [true, false], control: { type: 'select' } },
```

Comment:
I believe is better to use a `control: { type: 'boolean' }` rather than a select.

Reply:
done!